modules/cli/src/main/java/org/openapitools/openapistylevalidator/Main.java

```diff
@@ -30,8 +33,14 @@ public static void main(String[] args) {
 
         try {
             CommandLine commandLine = parser.parse(options, args);
-            Swagger swagger = new SwaggerParser().read(commandLine.getOptionValue(SOURCE_OPT_SHORT));
-            OpenApiSpecStyleValidator openApiSpecStyleValidator = new OpenApiSpecStyleValidator(swagger);
+            OpenAPIParser openApiParser = new OpenAPIParser();
+            ParseOptions parseOptions = new ParseOptions();
```

Comment:
I think you would like to resolve the spec that you are reading. See:
https://github.com/swagger-api/swagger-parser/issues/1149#issuecomment-510759906